mne/io/nirx/tests/test_nirx.py

```diff
@@ -79,6 +79,12 @@ def test_nirx_15_2_short():
     # Test data import
     assert raw._data.shape == (26, 145)
     assert raw.info['sfreq'] == 12.5
+    assert raw.info['meas_date'].month == 8
+    assert raw.info['meas_date'].day == 23
+    assert raw.info['meas_date'].year == 2019
+    assert raw.info['meas_date'].hour == 7
+    assert raw.info['meas_date'].minute == 37
+    assert raw.info['meas_date'].second == 4
```

Comment:
I would do it with == after creating a datetime instance with proper timezone. It's less much lines.